### ISSUE TITLE
Accuracy problem 

### ISSUE THREAD

**Issue by torrentillo** [MITESHPUTHRANNEU/Speech-Emotion-Analyzer#27]
Hi Mitesh, 
I’m trying to obtain the 70% accuracy you got but I’m only getting a 35% could you please tell me the database and send me the  exact code you used to get the 70%?? 
Thank you very much.
My email is listor69@gmail.com

**Comment by MITESHPUTHRANNEU** [comment]
Hello @torrentillo, this is the exact code that I had used to train my model. In order to achieve higher accuracy, there are 3 things that come to my mind. 
1. Try playing with the parameters while building the model and test it out.
2. Extract more features from the audio files. This can be done by increasing the sampling rate.
3. Add more data by copying the existing files. (Not the best way, but can improve the accuracy)

Hope this helps.

**Comment by torrentillo** [comment]
Hi Mites,

I used the exact code and I can't get the accuracy you got, what about the dataset you used? could you please tell me what it was?

**Comment by torrentillo** [comment]
I'm obtaining a lot of features for my audios with the value NA, do you know why  could be  this ?

**Comment by MITESHPUTHRANNEU** [comment]
I had used the dataset from the links I posted in the README file.

**Comment by torrentillo** [comment]
Yes, but when you filter only by 5 emotions you have like 1300 audios for training instead of 900 that I have if I do the same, so you've trained your model with some extra data that is not in the like you posted in the README file. 

Thank you

**Comment by beachboysqq** [comment]
Hi,I am trying using different models and increase the dropout value.But it seems still overfit. I can get a hign acc on training but low acc on test. Plz give me some suggestions~ many thanks~

**Comment by torrentillo** [comment]
Hi beach, 

I've achieved a 70 % accuracy with this model :

Ridge = sklearn.linear_model.Ridge()

**Comment by nikhil2593** [comment]
same..not getting specified accuracy with the provided model architecture and dataset.
